test/tests/positions/transformed_positions.i

```diff
@@ -0,0 +1,51 @@
+[Mesh]
+  [cmg]
+    type = CartesianMeshGenerator
+    dx = 1
+    dim = 1
+  []
+[]
+
+[Positions]
+  [input]
+    type = InputPositions
+    positions = '0 0 0
+                 1 0 0
+                 0 1 0'
+    outputs = none
+  []
+  [scale]
+    type = TransformedPositions
+    base_positions = 'input'
+    vector_value = '1 2 3'
+    transform = 'SCALE'
+  []
+  [rotate]
+    type = TransformedPositions
+    base_positions = 'input'
+    vector_value = '0 0 90'
+    transform = 'ROTATE_XYZ'
+  []
+  [translate]
+    type = TransformedPositions
+    base_positions = 'input'
```

Comment:
I assume that you could also supply `scale` or `rotate` either?

Reply:
I ll try. This would be legitimate use

Reply:
works as expected. 
Anything defined first is fine until we do dependency resolution at construction